ingestion/src/metadata/ingestion/ometa/ometa_api.py

```diff
@@ -415,13 +415,25 @@ def _get(
                 )
             return entity(**resp)
         except APIError as err:
-            logger.error(
-                "GET %s for %s." "Error %s - %s",
-                entity.__name__,
-                path,
-                err.status_code,
-                err,
-            )
+            if err.status_code == 404 and entity.__name__ == "DatabaseService":
```

Comment:
Hi @ulixius9, let's try not to add any entity specifics in the API directly. The `OpenMetadata` client should be as generic as possible, so we should not add any reference to `DatabaseService`.

Also, note that the creation of the service is not happening in this function. the `_get` is run for any call to get_by_name or get_by_id. 

What I'd suggest is the following:
- In this function, instead of directly calling `logger.error`, return a `logger.info` if the status is 404, as not finding an entity is a usual scenario.
- If we have an APIError with any other status, then log the error as we are currently doing.

Then, improve the logging in [this](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/utils/helpers.py#L55) function, where we either GET the service or create a new one

Reply:
let me know if there are any questions